src/actions/popUpStateMachine.ts

```diff
@@ -0,0 +1,15 @@
+export const SHOW_POP_UP = "SHOW_POP_UP";
+export const HIDE_POP_UP = "HIDE_POP_UP";
+
+export const showPopUp = (stateMachine) => {
+    return {
+        type: SHOW_POP_UP,
+        stateMachine
```

Comment:
like the properties panel in XCStudio?
I would call these actions SHOW_STATEMACHINE_PROPERTIES / HIDE_STATEMACHINE_PROPERTIES.

So, 1. we can understand what we'll show from reading the action, 2. we can change the *way* we display things in the future, without renaming lots of actions.